tests/test_execute.py

```diff
@@ -376,7 +371,7 @@ def test_output_mimetype(self):
         """
         my_process = create_mimetype_process()
         service = Service(processes=[my_process])
-        self.assertEqual(len(service.processes.keys()), 1)
+        self.assertEqual(len(list(service.processes.keys())), 1)
```

Comment:
could be replaced with `len(service.processes)`

Reply:
done.